requirements.in

```diff
@@ -48,5 +48,6 @@ requests
 retrying
 six
 SQLAlchemy-Utils
+sqlalchemy < 1.4.0 # ImportError: cannot import name '_ColumnEntity' https://github.com/sqlalchemy/sqlalchemy/issues/6226
```

Comment:
Based on the ReadTheDocs error, I think you might need to add this to `requirements-docs.in` as well. I'm guessing it was being pulled in transitively with no restriction.

Reply:
good observation; so glad we have the doc build check!